### ISSUE TITLE
[Feature Request] Add an updater to the app

### ISSUE THREAD

**Issue by micah686** [Yink/Amadeus#45]
It would be nice if the app could check for updates, then either prompt you to download the latest .apk, or automatically download and install the newest update. It would be a lot more convenient than checking the github page every now and then, checking if a new build has come out. I would appreciate it if someone could work on this functionality in the near future.

**Comment by RIP95** [comment]
Yeah, that's a good idea, thought about it too. Will try implementing it in some way.

**Comment by 8BitAsian** [comment]
If you want, I can add a mailing list to the site so I can send everyone an email notification when a new build it out.
